### PR TITLE
chore: bump memory limit for CDK's serverless functions

### PR DESCRIPTION
**Pull request by BRKalow**
## 🔗 Relevant links

- [Preview link](https://dev-portal-git-BRANCH_NAME-hashicorp.vercel.app/PATH_TO_VIEW) 🔎
- [Asana task](url) 🎟️

## 🗒️ What

TF CDK has some large pages, which are exceeding the memory limits of our serverless functions and causing rendering to fail. I'm bumping up the memory limit here for those functions to see if we can get those pages rendering.

### CODE REVIEW

File: vercel.json [hunk 1]
```diff
@@ -1,5 +1,10 @@
 {
 	"github": {
 		"silent": false
+	},
+	"functions": {
+		"src/pages/terraform/cdktf/**": {
+			"memory": 2048
+		}
```
Comment (thiskevinwang) [inline vercel.json:8]: Docs for posterity
https://vercel.com/docs/project-configuration#project-configuration/functions/value-definition

**Review by thiskevinwang — APPROVED** [review]
👍

**Comment by ansgarm** [comment]
@mutahhir just noticed this today 😁 Can we get this merged? Especially now that we have the note on our CDKTF docs pointing to the new portal? 😇

**Comment by mutahhir** [comment]
It seems like the issue was resolved in some other way as I remember previously viewing the CDKTF API reference would not load. It's pretty spiffy now.

**Comment by BRKalow** [comment]
@mutahhir Yeah! We switched to rendering these pages at build time so we should be all set here 🤞

**Pull request closed without merging**